python-backend/app/api/users/minespace/models/minespace_user.py

```diff
@@ -41,7 +33,7 @@ def find_by_guid(cls, user_guid):
 
     @classmethod
     def find_by_email(cls, email):
-        return cls.query.filter_by(email=email).filter_by(deleted_ind=False).first()
+        return cls.query.filter_by(email=email).filter_by(deleted_ind=False).all()
```

Comment:
Are you sure the consuming code expects a collection, and not only one object?

Reply:
I made that change to simplify the resource, but this is bad because it's a unique column.  i'll change the resource